docs/ref/utils.txt

```diff
@@ -151,9 +151,11 @@ The functions defined in this module share the following properties:
 .. module:: django.utils.decorators
     :synopsis: Functions that help with creating decorators for views.
 
-.. function:: method_decorator(decorator)
+.. function:: method_decorator(decorator, name="")
 
-    Converts a function decorator into a method decorator. See :ref:`decorating
```

Comment:
Need a `.. versionchanged:: 1.9` annotation for the affected docs (here in and in CBGVs)